js/controller.js

```diff
@@ -256,19 +256,23 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onVcVideoElementCreated: function(event, params) {
       var videoElement = params.videoElement;
-      videoElement = this.findMainVideoElement(videoElement);
+      if (params.videoId === OO.VIDEO.MAIN) {
+        this.state.mainVideoElement = videoElement;
+      }
 
+      videoElement = this.findMainVideoElement(videoElement);
       //add loadedmetadata event listener to main video element
       if (videoElement) {
         videoElement.addEventListener("loadedmetadata", this.metaDataLoaded.bind(this));
-      }
-
-      if (Utils.isIE10()) {
-        videoElement.attr("controls", "controls");
+        if (Utils.isIE10()) {
+          videoElement.attr("controls", "controls");
+        }
       }
 
       if (params.videoId === OO.VIDEO.MAIN) {
-        this.state.mainVideoElement = videoElement;
+        if (videoElement) {
+          this.state.mainVideoElement = videoElement;
+        }
```

Comment:
No, we want to save initial value of mainVideoElement because on line 484 this value will be reused in the next attempt to obtain mainVideoElement